MaxText/configs/base.yml

```diff
@@ -81,7 +81,7 @@ logits_dot_in_fp32: True  # whether to use fp32 in logits_dense or shared_embedd
 remat_policy: 'full'
 scan_layers: True
 param_scan_axis: 1
-attention: 'flash' # Supported attention: dot_product, flash, gpu_flash_xla, gpu_flash_triton
+attention: 'dot_product' # Supported attention: dot_product, flash, gpu_flash_xla, gpu_flash_triton, cudnn_flash_te
```

Comment:
Let's remove gpu_flash_xla and gpu_flash_triton for now dispatch to the cudnn_flash_te codepath based on hardware
https://github.com/google/maxtext/blob/main/MaxText/configs/base.yml#L99

Reply:
I had a small doubt- as with gpu, someone would want use mha as well, is it really necessary to change it to flash attention based on hw?